README.md

```diff
@@ -9,12 +9,12 @@ IX is licensed under an MIT-style license.  See LICENSE.
 
 ## Requirements
 
-IX requires Intel DPDK and a supported Intel NIC: 
```

Comment:
It is difficult to follow actual changes when they are intermingled with white space fixes. White space fixes should be in a commit by themselves and only themselves.